core-chart/templates/postcreatehooks/kubestellar-controller.yaml

```diff
@@ -2,167 +2,11 @@
 apiVersion: tenancy.kflex.kubestellar.org/v1alpha1
 kind: PostCreateHook
 metadata:
-  name: wds
+  name: kubestellar-controller
```

Comment:
KubeStellar has many controllers.

The thing you mean here is "kubestellar-controller-manager".